tests/spread/plugins/v1/dotnet/snaps/dotnet-hello/snap/snapcraft.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   This is a sample dotnet 2.0.0 application
 
 grade: devel
-base: core16
+base: core18
```

Comment:
dotnet snap does not support core18

Reply:
It built & ran for me just fine?

Reply:
Confirmed again, seems fine..

Reply:
There's also a stage-package handling for core18 in the plugin..